examples/image_classification/ImageClassificationOpenCVRunner.js

```diff
@@ -0,0 +1,25 @@
+class ImageClassificationOpenCVRunner extends OpenCVRunner {
+  constructor() {
+    super();
+    this._labels = null;
+  }
+
+  _setLabels = (labels) => {
+    this._labels = labels;
+  };
+
+  _getLabels = async (url) => {
+    const result = await this._loadURL(url);
+    this._setLabels(result.split('\n'));
+    console.log(`labels: ${this._labels}`);
+  };
+
+  _getOtherResources = async () => {
+    await this._getLabels(this._currentModelInfo.labelsFile);
+  };
```

Comment:
Can we consolidate the above three functions with `ImageClassificationWebNNRunner`?